yarn/src/main/scala/org/apache/spark/deploy/yarn/Client.scala

```diff
@@ -912,7 +943,10 @@ object Client extends Logging {
   val ENV_DIST_CLASSPATH = "SPARK_DIST_CLASSPATH"
 
   // Subdirectory where the user's hadoop config files will be placed.
-  val LOCALIZED_HADOOP_CONF_DIR = "__hadoop_conf__"
+  val LOCALIZED_CONF_DIR = "__hadoop_conf__"
```

Comment:
why did you change the name of this?  please revert as previous name was more descriptive and the change is unnecessary.